.github/workflows/publish-snap.yaml

```diff
@@ -0,0 +1,26 @@
+name: publish-snap
+on:
+  release:
+    types: [published]
```

Comment:
Let's do the same thing that we're doing on https://github.com/PRQL/prql/pull/1883 — have it build on this PR so we can test it, and then when we can see it works, we'll move to `release.yaml`

Reply:
Yeah, sounds good.